robyn/scaffold/sqlalchemy/app.py

```diff
@@ -0,0 +1,32 @@
+from robyn import Robyn
+from sqlalchemy import create_engine, Column, Integer, String, Boolean
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+
+engine = create_engine("sqlite+pysqlite:///:memory:", echo=True)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+app = Robyn(__file__)
+
+
+@app.get("/")
+def index():
+    return "Hello World!"
+
+
+if __name__ == "__main__":
+    # create a configured "Session" class
+    Base = declarative_base()
```

Comment:
Maybe it's better to move the sqlalchemy stuff to another module? models.py perhaps?